import_export/export-wordpress.py

```diff
@@ -77,7 +78,10 @@ def update_wordpress(id, language, title, content, excerpt=""):
   url = f"{export_url}/wp-json/elsa/v1/translate/{id}/{language}"
   user = os.environ.get('WP_EXPORT_USER')
   password = os.environ.get('WP_EXPORT_PASSWORD')
-  response = requests.post(url, headers={'User-Agent': ""}, auth=HTTPBasicAuth(user, password), json=message)
+
+  # response = requests.post(url, headers=header, json=message)
+  response = requests.post(url, headers={"Content-Type": "application/json", 'User-Agent': ""}, auth=HTTPBasicAuth(user, password), json=message)
+  print(user, password, response.text)
```

Comment:
can we remove username and password from this, don't want to have sensitive things in logs

Reply:
Oop let me change this in Master quick - sorry!